ci-operator/step-registry/ipi/conf/ipi-conf-commands.sh

```diff
@@ -0,0 +1,201 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wait; fi' TERM
+
+cluster_profile=/var/run/secrets/ci.openshift.io/cluster-profile
+cluster_name=${NAMESPACE}-${JOB_NAME_HASH}
```

Comment:
nit: Neither of these is likely to contain shell-sensitive characters, but I like quoting variables just in case:

```sh
cluster_name="${NAMESPACE}-${JOB_NAME_HASH}"
```

It's not clear to me how [ShellCheck passed][1] without that.  Is the ShellCheck job currently an empty shim?

[1]: https://prow.svc.ci.openshift.org/view/gcs/origin-ci-test/pr-logs/pull/openshift_release/6708/pull-ci-openshift-release-master-step-registry-shellcheck/1842

Reply:
There's no need to quote assignments, there's no ambiguity (as opposed to expansion in commands):

```sh
$ a='a b'; b='c d'; c=$a-$b; echo "==$c=="
==a b-c d==
```